scipy/stats/tests/test_boost_ufuncs.py

```diff
@@ -1,7 +1,7 @@
 import pytest
 import numpy as np
 from numpy.testing import assert_allclose
-from scipy.stats import _boost
+import scipy.special._ufuncs as scu
```

Comment:
Minor query but should these tests be under stats if they only test special functions?

Reply:
Addressed. I added the tests in this file to `test_ufunc_signatures.py`. It felt redundant to have a separate file when `_ufuncs` are already tested somewhere. Let me know if semantics of the change look good to you. I will make necessary changes if opinions differ.

Reply:
Honestly, I don't think these tests are a good fit for `test_ufunc_signatures.py`, since they also test accuracy. The intention of `test_ufunc_signatures` is just to make sure that we have `float32` preserving versions for every signature. Almost everything in special is a `ufunc`, so it's fine to have multiple test files for `ufunc`s.

Reply:
Addressed. Please check if the changes look good to you now.